marketplace/netlify-build/src/app/index.js

```diff
@@ -64,14 +64,17 @@ export default class NetlifyAppConfig extends React.Component {
 
     const ticketId = await NetlifyClient.createTicket();
 
-    this.setState({
-      ready: true,
-      config,
-      enabledContentTypes,
-      contentTypes: contentTypesResponse.items.map(ct => [ct.sys.id, ct.name]),
-      netlifySites: uniqBy(netlifySites, s => s.id),
-      ticketId
-    });
+    this.setState(
+      {
+        ready: true,
+        config,
+        enabledContentTypes,
+        contentTypes: contentTypesResponse.items.map(ct => [ct.sys.id, ct.name]),
+        netlifySites: uniqBy(netlifySites, s => s.id),
+        ticketId
+      },
+      app.setReady
```

Comment:
I'd wrap with a function

Reply:
any reason to go for the more verbose variation?

Reply:
so we make sure it's called with 0 args (just to be safe)